pyproject.toml

```diff
@@ -24,6 +24,7 @@ holidays = "^0.11.2"
 email-validator = "^1.1.3"
 zbar-py = "^1.0.4"
 opencv-python-headless = "^4.5.3"
+beautifulsoup4 = "^4.10.0"
```

Comment:
Yoh, haven't seen this in a while

Reply:
Yeah unfortunately we have to scrape, and this is the best library I know for scraping :)